src/entry.rs

```diff
@@ -309,6 +320,29 @@ impl<'a> EntryFields<'a> {
                                         for `{}`", mode, dst.display()), e)
             }));
         }
+        // Windows does not completely support posix xattrs
+        // https://en.wikipedia.org/wiki/Extended_file_attributes#Windows_NT
+        if cfg!(target_family = "unix") && preserve_xattrs {
```

Comment:
Ah ok, then in that case we should probably follow the same behavior as `tar` and not do it by default. I'd personally prefer to avoid `unpack` and `unpack_foo` method duplication, so perhaps an option could be set somewhere? Like maybe `Archive::set_unpack_xattrs` which just sets a boolean flag.

We'd probably also want it to be a unix-specific extension trait, but I can take care of that part as well.